### ISSUE TITLE
Auto Refresh at launch/ Shortcuts functionality

### ISSUE THREAD

**Issue by XxBlackHatxX** [etesync/ios#36]
I know because of iOS limitations that you were not able to run the app in the background. Currently on iOS 13/14 you can have automation in the shortcut app. I created shortcut that will show a notification asking to if I would like to refresh when selected it will open EteSync. Then I manually click the refresh button and go back to my application.

It would be great to have a simple shortcut integration that when called will launch EteSync to open and immediately sync every time. I hope that made sense. For example my VPN app has a shortcut integration that when you call their auto connect integration will automatically launch their VPN application and immediately connect to a server every time it’s called. I hope that wouldn’t be too default to do. If so another option would be to add in the app setting itself an auto sync so every time you open the app it will start syncing without needing to press the button first. 

If you were able to do one of those two things that would make that would make the workflow much smoother.

**Comment by aliktb** [comment]
I'm no iOS developer, so I don't know how easy it would be but perhaps with iOS 14 a home screen widget could perform a manual sync. It's not exactly a smart solution as it requires the user to manually tap on it every time to sync. However, I find a larger 2x2 widget is easier to find than an app which inevitably ends up buried in a folder or in the App drawer/library. I have no idea if it would be easy to implement or if it is even possible but it's an idea I've had for a while.

**Comment by catwithbanana** [comment]
I agree that this would be a _huge_ functionality improvement - as well as a clever way to work around the limitations of the platform.
